code/datums/dog_fashion.dm

```diff
@@ -10,6 +10,8 @@
 	// sprite that the associated item uses
 	var/icon_file
 	var/icon_state
+	var/alpha
+	var/color
```

Comment:
actually how does this compile BYOND should be throwing a fit due to reserved var names